tutorials/actors_demo/actor_demo.md

```diff
@@ -0,0 +1,147 @@
+# Actors demo
+
+In this tutorial we will learn how to add `actors` to our world and how to create a scripted animation. You can find the final world of this tutorial [here](actor_demo.sdf)
+
+Animations are very useful if we want to have entities following a predefined path in simulation without being affected by the physics. This means that they won't fall due to gravity or collide with other objects. They will however, have a 3D visualization which can be seen by RGB cameras, and 3D meshes which can be detected by GPU based sensors. The tutorial explains how to create open-loop trajectories which don't interact with the rest of the simulation.
+
+## Actors
+
+In Ignition Gazebo, an animated model is called an `actor`.
+
+There are two types of animations which can be used separately or combined together:
+
+* Skeleton animation, which is relative motion between links in one model:
+
+![skeleton_movement](skeleton_movement.gif)
+
+* Trajectory animation, which carries all of the actor's links around the world as one group along a trajectory:
+
+![trajectory_movement](trajectory_movement.gif)
+
+* Combined, to achieve a skeleton animation which moves in the world:
+
+![combined_movement](combined_movement.gif)
+
+Actors have the following properties:
+
+* No forces are applied on them, be it from gravity or contact or anything else
+* Actors support skeleton animation imported from COLLADA(.dae) and BVH(.bvh) files.
+* Actors can have trajectories scripted directly in SDF.
+
+You can check out the full specification for the `<actor>` SDF element [here](http://sdformat.org/spec?ver=1.7&elem=actor).
+
+## Skeleton
+
+Ignition gazebo supports two different skeleton animation file formats: COLLADA (.dae) and Biovision Hierarchy (.bvh).
+
+Open any world `empty.sdf` world, for example and add an actor called `actor_walking` as follows:
```

Comment:
To an end user, they may not know where to find `empty.sdf`. How about adding a step for them to download that world from GitHub first?